lib/dpl/provider/surge.rb

```diff
@@ -0,0 +1,33 @@
+module DPL
+  class Provider
+    class Surge < Provider
+      npm_g 'surge'
+
+      def project
+        File.expand_path("./" + (options[:project] || '') )
```

Comment:
Does this work if the working directory changes during the build? (I'm not familiar with `surge`, so this is a genuine question.)